src/librustc_mir/interpret/eval_context.rs

```diff
@@ -82,21 +85,22 @@ pub struct Frame<'mir, 'tcx, Tag=(), Extra=()> {
     ////////////////////////////////////////////////////////////////////////////////
     /// The block that is currently executed (or will be executed after the above call stacks
     /// return).
-    pub block: mir::BasicBlock,
+    /// If this is `None`, we are unwinding and this function doesn't need any clean-up.
+    /// Just continue the same as with
```

Comment:
```suggestion
    /// Just continue the same as with `Resume`.
```
Seems like I somehow fell asleep half-way through writing this comment, or so? ;)